Components/Windows/create-local-user/create-local-user.yaml

```diff
@@ -0,0 +1,112 @@
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this
+# software and associated documentation files (the "Software"), to deal in the Software
+# without restriction, including without limitation the rights to use, copy, modify,
+# merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+# INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+# PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+# HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+# OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+name: LocalUser
+description: 'Creates a local Windows User account without admin permission but RDP login'
+schemaVersion: 1.0
+constants:
+  - Username:
+      type: string
+      value: dba-access
+  - FullName:
+      type: string
+      value: DBA access
+  - Description:
+      type: string
+      value: Local user account for DBA Jump host access
+  - PasswordSecret:
+      type: string
+      value: Jumphost/RdpUser
+phases:
+  - name: build
+    steps:
+      - name: PrepareAwsPSTools
+        action: ExecutePowerShell
+        inputs:
+          commands:
+            - |
+              $ErrorActionPreference = 'Stop'
+              $ProgressPreference = 'SilentlyContinue'
+            
+              Write-Host "Checking if NuGet Package Provider is available..."
+              $availableProviders = Get-PackageProvider -ListAvailable
+              
+              if ( ($availableProviders.Name -Like "NuGet").Length -Eq 0 ) {
+              
+                Write-Host "NuGet is not available. Installing it."
+                Install-PackageProvider -Name NuGet -Force
+              }
+              Write-Host "Done."
+              
+              Write-Host "Checking if AWS PowerShell Tools are available..."
+              $availableModules = Get-Module -ListAvailable
+              
+              if ( ($availableModules.Name -Like "AWSPowerShell").Length -Eq 0 ) {
+              
+                Write-Host "AWS PowerShell Tools are not available. Installing them."
+                Install-Module -Name Aws.Tools.Installer -Force
+              }
+              Write-Host "Done."
+              
+      - name: CreateLocalUser
+        action: ExecutePowerShell
+        inputs:
+          commands:
+            - |
+              $ErrorActionPreference = 'Stop'
+              $ProgressPreference = 'SilentlyContinue'
+              
+              Write-Host "Fetching password from Secrets Manager..."
+              $secret = Get-SECSecretValue -SecretId "{{ PasswordSecret }}"
+              $secretValue = ConvertFrom-JSON $secret.SecretString
```

Comment:
Can we use the parameter in here please?

```ConvertFrom-Json -InputObject $secret.SecretString```

Reply:
done